src/ConnectedMode/ServerSentEvents/Issue/IssueServerEventsListener.cs

```diff
@@ -72,8 +76,12 @@ public async Task ListenAsync()
                         return;
                     }
 
-                    // TODO: Implement handling of event.
                     logger.LogVerbose(Resources.Suppression_IssueChangedEventReceived, issueServerEvent);
+                    foreach(var item in issueServerEvent.BranchAndIssueKeys)
+                    {
+                        // TODO - check branch handling. Are issueKeys unique for each branch?
+                        serverIssuesStoreWriter.UpdateIssue(item.IssueKey, issueServerEvent.IsResolved);
```

Comment:
This is a simple implementation so we can do some testing with SonarQube.

We might need to change the writer API to handle bulk changes.